src/components/snackbar/snackbar.css

```diff
@@ -0,0 +1,56 @@
+.snackbar {
+  position: fixed;
+  left: 50%;
+  top: 10%;
+  transform: translate(-50%, -50%);
+  width: 300px;
+  height: 80px;
+  border-radius: 8px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  background-color: rgba(0, 245, 147, 0.8);
+}
+
+.snackbar.symbol {
+  flex: 20%;
+}
+
+.snackbar.message {
+  flex: 80%;
+}
```

Comment:
I don't think this does what you think it does. 

I think you're trying to target the child with the `.message` class from the snackbar. 

What this actually does is target a component that has both `snackbar` _and_ `message` classes. 

Have a little peek at the way [CSS class selectors](https://developer.mozilla.org/en-US/docs/Web/CSS/Class_selectors) work for more info.

Reply:
Added spaces where required so the child is targeted as intended.